engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -493,6 +514,20 @@ public void setErrorMessage(String errorMessage)
       this.errorMessage = errorMessage;
     }
 
+    public String getLogFileName()
```

Comment:
@vrozov  introducing a common parent doesn't look feasible as java doesn't support extending a class from multiple parents. And as I mentioned before having the field in StramEvent will affect almost 14-15 classes which are less likely to have this field in future.